lib/floe/workflow/intrinsic_function.rb

```diff
@@ -1,7 +1,24 @@
 module Floe
   class Workflow
-    module IntrinsicFunction
-      def self.evaluate(payload, context = {}, input = {})
+    class IntrinsicFunction
+      def self.value(payload, context = {}, input = {})
+        new(payload).value(context, input)
+      end
+
+      def self.intrinsic_function?(payload)
+        Parser.new.parse(payload)
+        true
+      rescue Parslet::ParseFailed
+        false
+      end
```

Comment:
@Fryguy not sure if you had a better idea for identifying an IntrinsicFunction

Reply:
Yeah, for now I think it's simpler to do `payload.starts_with?("States.")`

Reply:
```suggestion
      def self.intrinsic_function?(payload)
        payload.start_with?("States.")
      end
```

Reply:
Can you also add a spec for this method to the intrinsic_function_spec.rb?